hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -81,6 +81,13 @@
    */
   public static final String PREFETCH_BLOCKS_ON_OPEN_KEY = "hbase.rs.prefetchblocksonopen";
 
+  /**
+   * Configuration key to cache blocks when a compacted file is written, predicated on prefetching
+   * being enabled for the column family.
+   */
+  public static final String PREFETCH_COMPACTED_BLOCKS_ON_WRITE_KEY =
```

Comment:
A bit confusing.. Are we doing the prefetch of the new compacted file once it is written?
Dont think so.. When we write the file, that time itself the caching happens. So it is cache on write. Why its called prefetch then?  There is no extra fetch op happening right?